app/views/questions/show.html.haml

```diff
@@ -1,90 +1,91 @@
-    %meta{:charset => "utf-8"}/
-    %meta{:content => "width=device-width, initial-scale=1", :name => "viewport"}/
-    %link{:href => "http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css", :rel => "stylesheet"}/
-    %script{:src => "https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"}
-    %script{:src => "http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"}
-  %body
-    .container
-      .panel.panel-info
-        .panel-heading
-          .row
-            .col-sm-4
-              %td= link_to "Back", questions_path, class: "btn-sm btn-info"
-            .col-sm-8
-              = @question.title
-        .panel-body
-          .row
-            .col-sm-2
-              %button#testBtn.btn.btn-success.glyphicon.glyphicon-thumbs-up{"data-loading-text" => " ... ", :type => "button"}
-                4
-              %button#testBtnDown.btn.btn-success.glyphicon.glyphicon-thumbs-down{"data-loading-text" => " ... ", :type => "button"}
-                4
-            .col-sm-8
-              %p= @question.body
-            .col-sm-2
-              %label{:for => "sel"} Flag:
-              %select#sel.form-control
-                %option Duplicate
-                %option Out of topic
-                %option Contains inappropriate language
+= render partial: 'layouts/breadcrumb'
+%h1
+  =@question.title
 
-          .container
-            .row
-              .col-sm-1
-                / Button trigger modal
-                %button.btn.btn-primary.btn-md{"data-target" => "#myModalHorizontal", "data-toggle" => "modal"}
-                  Respond
-                / Modal
-                #myModalHorizontal.modal.fade{"aria-hidden" => "true", "aria-labelledby" => "myModalLabel", :role => "dialog", :tabindex => "-1"}
-                  .modal-dialog
-                    .modal-content
-                      / Modal Header
-                      .modal-header
-                        %button.close{"data-dismiss" => "modal", :type => "button"}
-                          %span{"aria-hidden" => "true"} ×
-                          %span.sr-only Cancel
-                        %h4#myModalLabel.modal-title
-                          Your Response
-                      / Modal Body
-                      .modal-body
-                        %form.form-horizontal{:role => "form"}
-                          %textarea#comment.form-control{:rows => "5"}
-                      / Modal Footer
-                      .modal-footer
-                        %button.btn.btn-default{"data-dismiss" => "modal", :type => "button"}
-                          Cancel
-                        %button.btn.btn-primary{:type => "button"}
-                          Submit
-              .col-sm-1
-                %td= link_to "Edit", edit_question_path(@question), class: "btn btn-info"
-                -#%input.btn-info.btn-info{:type => "submit", :value => "Delete"}/
+%h3
+  = "Exercise: #{@question.exercise_id}"
```

Comment:
I tried to change the line to @question.exercise.name and @question.exercise_id.name both broke. I am not how to show the exercise name.